src/my-workspace/views/Collections.tsx

```diff
@@ -1,11 +1,16 @@
-import React, { Fragment, FunctionComponent } from 'react';
```

Comment:
Don't mind this file. This is so you can test the component under "Mijn werkruimte". I'll remove before merge.